src/style.css

```diff
@@ -0,0 +1,70 @@
+html {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-size: 12px;
+  font-weight: 500;
+  background-color: #eee;
+}
+
+.header {
+  background-color: rgba(255, 255, 255, 1);
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+  
+  height: 60px;
+  margin-top: 20px;
+  margin-left: 20px;
```

Comment:
1) write code according to design, there is no those margins on design
2) don't add margins to block (BEM)